pkg/controller/kibana/stackmon/__snapshots__/sidecar_test.snap

```diff
@@ -849,231 +849,3 @@ processors:
     - add_host_metadata: null
 
 ---
-
```

Comment:
This change threw me for a loop. Initially when updating snaps the mountpath for this test changed, and I couldn't figure out why for the longest time. Then I noticed that this snap is referencing a test that doesn't exist any longer, hence it being removed.